### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by Git-HorizonZz**
-remove alumni & movers
-add new members
-update zane's photo

### CODE REVIEW

**Comment by Git-HorizonZz** [comment]
New organization:
1.  Captains
2. 2023
3. 2024
4. 2025
5. 2026
Within each group the order is:
1. Media
2. Pneumatics
3. Programming
4. Electrics
5. Mechanics

File: _pages/team.html [hunk 1]
```diff
@@ -10,232 +10,232 @@
...
+            <p class="label">📸 Media 📸</p>
+            <h5>Sophia Gavasheli '23</h5>
+            <p>Sophia is a second year programmer.</p>
         </div>
         <div>
             <img src="/images/team/alexander-perez.png">
             <p class="label">💨 Pneumatics 💨</p>
             <h5>Alexander Perez '23</h5>
-            <p>Alexander is a second year pneumaticaneer.</p>
+            <p>Alexander is a third year pneumaticaneer.</p>
```
Comment (ArchdukeTim) [inline _pages/team.html:46]: pneumaticist?

Comment (Git-HorizonZz) [reply _pages/team.html:46]: 'pneumaticaneer' was the word used on the website previously, so I kept it for continuity. Also a bonus is that when searching 'pneumaticaneer' we are the only result lol

**Review by ErikBoesen — APPROVED** [review]

File: _pages/team.html (hunk 1 above)
Comment (ArchdukeTim) [reply _pages/team.html:46]: lol

**Pull request merged**